lib/min_heap.rb

```diff
@@ -8,27 +8,30 @@ def initialize(key, value)
 end
 
 class MinHeap
-
   def initialize
     @store = []
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(log n)
+  # Space Complexity: O(n)
```

Comment:
Since you are only adding one item to the heap and use an iterative `heap_up` method, this is actually constant.